modern/src/App.js

```diff
@@ -21,36 +22,59 @@ async function getSkin() {
   return await initialize(zip, skinXml);
 }
 
+function handleMouseEventDispatch(node, event, eventName) {
+  const rect = event.target.getBoundingClientRect();
+  const x = event.clientX - rect.left;
+  const y = event.clientY - rect.top;
+  node.js_trigger(eventName, x, y);
+}
+
+function handleMouseButtonEventDispatch(
+  node,
+  event,
+  leftEventName,
+  rightEventName
+) {
+  if (event.button === 2) {
```

Comment:
Rather than having an if/else with very similar function calls, could we do:

```
const eventName = event.button === 2 ? rightEventName : leftEventName;
handleMluseEventDispatch(node, event, eventName);
```

Reply:
yup, nice and succinct